### PR TITLE
Fix some small details after removing Http.Context

### PR DESCRIPTION
**Pull request by marcospereira**
This is a follow up to #9320.  See the conversation there.

### CODE REVIEW

File: documentation/manual/releases/release28/migration28/Migration28.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ xxx
 
 #### Java API
 
-1. xxx
+1. In Play 2.7 we deprecate `play.mvc.Http.Context` in favor of directly using `play.mvc.Http.RequestHeader` or `play.mvc.Http.RequestHeader`. We have now removed `Http.Context` and if your application was still depending on it, you should read [[Play 2.7 migration guide instructions|JavaHttpContextMigration27]].
```
Comment (mkurz) [inline documentation/manual/releases/release28/migration28/Migration28.md]: `play.mvc.Http.RequestHeader` or `play.mvc.Http.RequestHeader`?

Comment (dwijnand) [reply documentation/manual/releases/release28/migration28/Migration28.md]: Yes.

**Comment by dwijnand** [comment]
Needs a rebase (and a commit squash).

**Comment by marcospereira** [comment]
Rebased and squashed. :-)

**Pull request merged**